utopia-remix/app/routes/projects.tsx

```diff
@@ -724,7 +726,14 @@ const ProjectCard = React.memo(
           }}
         >
           <div style={{ display: 'flex', flexDirection: 'column', padding: 10, gap: 5, flex: 1 }}>
-            <div style={{ fontWeight: 600 }}>{projectTitle}</div>
+            <div style={{ fontWeight: 600, display: 'flex', gap: '10px', alignItems: 'center' }}>
+              <span>{projectTitle}</span>
+              <ProjectBadge
+                accessLevel={
+                  (project.ProjectAccess?.access_level as AccessLevel) || AccessLevel.PRIVATE
```

Comment:
these `as AccessLevel` look pretty suspicious to me, especially because they're a recipe for issues if we have a discrepancy between the data on the DB and the enum. I would instead suggest to have a helper function that gets the right `AccessLevel` (with a switch/assertNever) given a number